Content.Server/DeltaV/Administration/Commands/LoadCharacter.cs

```diff
@@ -0,0 +1,169 @@
+using System.Linq;
+using Content.Server.Administration;
+using Content.Server.GameTicking;
+using Content.Server.Players;
+using Content.Server.Preferences.Managers;
+using Content.Server.Station.Systems;
+using Content.Shared.Administration;
+using Content.Shared.Humanoid;
+using Content.Shared.Humanoid.Prototypes;
+using Content.Shared.Preferences;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using Robust.Shared.Network;
+using Robust.Shared.Prototypes;
+
+// This literally only exists because haha felinid oni
```

Comment:
What is this talking about?

Reply:
The fact it really serves no purpose except to break stuff by loading characters onto another species. You could load a felinid character onto an oni and get a buff felinid, which was really funny